.pre-commit-config.yaml

```diff
@@ -1,8 +1,26 @@
 exclude: '^$'
 fail_fast: false
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v2.2.3
     hooks:
     - id: flake8
-      additional_dependencies: [flake8-docstrings, flake8-debugger, flake8-bugbear]
+      additional_dependencies: [flake8-docstrings, flake8-debugger, flake8-bugbear, mccabe]
+      args: [--max-complexity, "10"]
```

Comment:
Should these be also updated to `.pre-commit-config.yaml` so that they are run before a commit is allowed?

Reply:
This is .pre-commit-config.yaml

Reply:
:man_facepalming: